apps/microsoft-teams/frontend/src/components/config/ConfigPage/ConfigPage.tsx

```diff
@@ -46,13 +52,22 @@ const ConfigPage = () => {
 
   return (
     <>
-      <AccessSection tenantId={parameters.tenantId} dispatch={dispatchParameters} />
-      {isAppInstalled && (
+      <Box className={styles.box}>
+        <Heading>{headerSection.title}</Heading>
+        <Paragraph>{headerSection.description}</Paragraph>
+        <hr className={styles.splitter} />
+        <AccessSection
+          dispatch={dispatchParameters}
+          parameters={parameters}
+          isAppInstalled={isAppInstalled}
+        />
+      </Box>
+      {isAppInstalled && accounts.length ? (
```

Comment:
nit: if you have `isAppInstalled && accounts.length && (<NotificationsSection .... />)` here does this cause the weird issue of showing the "0"? I just tend to prefer avoiding a return of `null`

Reply:
Yes, it did cause the weird "0" issue, which is why I changed it from `isAppInstalled && accounts.length && (<NotificationsSection .... />)` to what I have here 😄